### ISSUE TITLE
QUESTION: How do I stop the extension from marking 'gl_VertexIndex' as an undeclared identifier?

### ISSUE THREAD

**Issue by Lexxaro** [danielscherzer/GLSL#122]
I'm using the extension for a vulkan project where I'm compiling all my shaders with a custom build tool using the glslc.exe that is  provided by the Vulkan SDK. This compiler doesn't know of 'gl_VertexID', so I'd like to use 'gl_VertexIndex'. 

For some reason the extension marks 'gl_VertexIndex' as an undeclared identifier. How could I change this behaviour?

**Comment by danielscherzer** [comment]
Thanks for using my extension!

`gl_VertexIndex` is known to the extension for coloring and for auto-completion.
To clarify: You mean the extension is putting wiggly lines below the `gl_VertexIndex` keyword and reporting an error regarding gl_VertexIndex' is an undeclared identifier?

If this is the case than the shader compiler is returning this error.
The extension uses the shader compiler of the driver of your graphics card as a default or you can set a custom one (including arguments) in the extension options.

**Comment by Lexxaro** [comment]
Yes, that's what I meant. Using the custom compiler option fixed it.

Thanks for the quick reply.
